src/main/java/org/craftercms/studio/model/rest/ApiResponse.java

```diff
@@ -99,11 +99,31 @@ public class ApiResponse {
             "Try pasting the content to a different folder", StringUtils.EMPTY);
     public static final ApiResponse BLOB_NOT_FOUND = new ApiResponse(7005, "Content not found in blob store",
             "Check your blob store configurations", StringUtils.EMPTY);
+    public static final ApiResponse CONTENT_IN_PUBLISH_QUEUE = new ApiResponse(7006, "Cannot edit content that is part of an active publish package",
+            "Cancel affected publish packages and retry", StringUtils.EMPTY);
+    public static final ApiResponse PUBLISHING_PACKAGE_NOT_FOUND = new ApiResponse(7007, "Publishing package not found",
+            "Check if you sent in the right Publishing Package Id", StringUtils.EMPTY);
+    public static final ApiResponse INVALID_PACKAGE_STATE = new ApiResponse(7008, "Requested package is not in the required state for this operation",
+            "Check the current package state", StringUtils.EMPTY);
+    public static final ApiResponse PACKAGE_ALREADY_APPROVED = new ApiResponse(7009, "Requested package is already APPROVED",
+            "Check the current package state", StringUtils.EMPTY);
 
     // 8000 - 9000
     public static final ApiResponse PUBLISHING_DISABLED = new ApiResponse(8000, "Publishing is disabled",
             "Advise the user to enable publishing", StringUtils.EMPTY);
 
+    public static final ApiResponse S3_UNREACHABLE = new ApiResponse(8001, "S3 is unreachable",
```

Comment:
Let's move these to an AWS section. We can start a new block if nothing already exists.